bundle/config/mutator/resourcemutator/configure_dashboards_serialized_dashboard.go

```diff
@@ -44,7 +44,7 @@ func (c configureDashboardSerializedDashboard) Apply(_ context.Context, b *bundl
 				return v, nil
 			}
 
-			contents, err := b.SyncRoot.ReadFile(path)
+			contents, err := b.BundleRoot.ReadFile(path)
```

Comment:
Can you add a comment explaining why BundleRoot is the correct one in this case?